apps/src/templates/manageStudents/ManageStudentsTable.jsx

```diff
@@ -371,4 +372,8 @@ export default connect(state => ({
   studentData: convertStudentDataToArray(state.manageStudents.studentData),
   editingData: state.manageStudents.editingData,
   addStatus: state.manageStudents.addStatus,
+}), dispatch => ({
+  saveAllStudents() {
+    dispatch(saveAllStudents());
+  },
 }))(ManageStudentsTable);
```

Comment:
This is a good place for the propsFromDispatch shorthand:
```js
export default connect(state => ({
  loginType: state.manageStudents.loginType,
  // Other propsFromState stuff...
}), {
  // If you just pass an object of action creators as the second argument to connect,
  // react-redux assumes they're all simple pass-throughs like this:
  // saveAllStudents: (...args) => dispatch(saveAllStudents(...args)),
  saveAllStudents,
})(ManageStudentsTable);
```

Reply:
I'm going to leave this one as it. I find it confusing to understand what's going without the explicit call.